src/actions/index.jsx

```diff
@@ -28,6 +59,9 @@ export function signInAPI() {
       .signInWithPopup(provider)
       .then((payload) => {
         dispatch(setUser(payload.user));
+        (async () => {
```

Comment:
Is this specifically for Facebook? If not we should rename this because I need to make a Issue and assign you or @ashwaryy to it where only you buys have access to my firebase and the firewall block me from Facebook and twitter. I need you guys to go get the developer tokens to set up the authentication buttons on the login page.

Reply:
I have to make sure all authentications execute these lines, not sure if its being done.